### PR TITLE
[CU-hf59gy] Update creator/update matches for single elimination tournament's type

### PR DESCRIPTION
**Pull request by Zummek**

### CODE REVIEW

**Comment by Zummek** [comment]
Task linked: [CU-hf59gy Update creator/update matches for single elimination tournament's type](https://app.clickup.com/t/hf59gy)

**Review by PrzemekRychter — APPROVED** [review]
Great! Before merging to main we should merge `[CU-hf59j3] Prepare database for split tournament ` to this branch because it is an extension of this branch.

File: server/database/repositories/TournamentRepository.ts [hunk 1]
```diff
@@ -42,8 +46,10 @@ function toTournamentDb(t: TournamentWithoutMS) : TournamentDb {
     teams: t.teams.map((team) => toTeamDb(team)),
     matches: t.matches.map((match) => toMatchDb(match)),
     isFinished: t.isFinished,
+    type: t.type,
   };
 }
+
 function toTournament(tDoc: TournamentDocument) : TournamentWithoutMS {
   return {
     id: tDoc._id.toString(),
```
Comment (eastemployeer) [inline server/database/repositories/TournamentRepository.ts:59]: Change this name to something more understandable `tDoc`

File: server/concepts/Tournament.ts [hunk 2]
```diff
@@ -10,6 +11,10 @@ import AppError from '../utils/appError';
 import Match from './Match';
 import MSOrganization from './MSOrganization';
 
+interface SingleEliminationCreatorMatch extends MatchWithoutMS {
```
Comment (eastemployeer) [inline server/concepts/Tournament.ts]:
```suggestion
interface SingleEliminationMatchCreator extends MatchWithoutMS {
```

File: server/database/repositories/TournamentRepository.ts (hunk 1 above)
Comment (Zummek) [reply server/database/repositories/TournamentRepository.ts:59]: This is code from the other task. I know these names aren't the best. I wrote about this. By the way of the correction task it should be changed @PrzemekRychter

**Pull request merged**